lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -20,6 +20,10 @@
 Applies DSC Resources by generating a configuration file and applying it.
 EOT
 
+  def vendored_modules_path
+    File.expand_path(Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/dsc_resources').gsub(/\//,'\\')
```

Comment:
It was necessary because i put backslashes in the erb template. Replacing those with forward slashes removes the need for it. Fixing now.